INSTALL.rst

```diff
@@ -339,7 +339,7 @@ before you get started.
   conda install -c conda-forge --file requirements/test.txt
   conda install -c conda-forge pre-commit
   # Install build dependencies of scikit-image
-  conda install -c conda-forge --file requirements/build.txt
+  pip install -r requirements/build.txt
```

Comment:
For some reason GitHub shows three other commits that have been merged as part of this pull request (see https://github.com/scikit-image/scikit-image/pull/6814/commits). I'm confused why they show up in the diff for this PR, because they are already merged. :thinking: 

@decorouz, I can check out your branch locally and remove them for you if you want, but let me know if you want to give it a shot yourself and I'll try to help with that. :wink:

Reply:
Yes, I would like to give it a shot myself.
I looked up stack-overflow for some pointers.

```
git checkout <branch>

git revert <commits> 

git push origin <branch> --force
```
Would this address the issue?

Reply:
I'd probably do it with the [`git rebase -i HEAD~9`](https://stackoverflow.com/a/11955118/8483989) and then `git push -f`.

Reply:
Thanks @lagru 
Following your advice, I was able to remove the confusing commits. 
I also updated the docstring to match [numpydoc style](https://numpydoc.readthedocs.io/en/latest/format.html#short-summary) closely.
Please review.

Reply:
Thank you @lagru.
That was some commits clean-up-job you did on my behalf.